.github/workflows/testing.yml

```diff
@@ -76,7 +76,7 @@ jobs:
   bazel:
     runs-on: ubuntu-latest
     env:
-      USE_BAZEL_VERSION: 5.0.0
+      USE_BAZEL_VERSION: 6.0.0
```

Comment:
Why is this still showing up? https://github.com/grpc/grpc-java/pull/10730 is merged, and the file is updated.

https://github.com/grpc/grpc-java/blob/3e3ba56838ddabb57a241718b568db61e11d92cc/.github/workflows/testing.yml#L76-L79

Reply:
Because it hasn't yet been merged in/rebased. The diff is based on the master commit I branched from. I was expecting it to still show up.

Reply:
Ah, ok. I thought it'll re-compare against current master.